yarn.lock

```diff
@@ -23533,6 +23533,13 @@ rxjs@6.6.2:
   dependencies:
     tslib "^1.9.0"
 
+rxjs@^6.5.2:
+  version "6.6.3"
+  resolved "https://registry.yarnpkg.com/rxjs/-/rxjs-6.6.3.tgz#8ca84635c4daa900c0d3967a6ee7ac60271ee552"
+  integrity sha512-trsQc+xYYXZ3urjOiJOuCOa5N3jAZ3eiSpQB5hIT8zGlL2QfnHLJ2r7GMkBGuIausdJN1OneaI6gQlsqNHHmZQ==
+  dependencies:
+    tslib "^1.9.0"
+
```

Comment:
Hmm, why is RxJs being updated?

Reply:
hymmm -- maybe related to:
https://github.com/grafana/grafana/issues/26748

somhow the yarn.lock versions are not respected

Reply:
Maybe something else refers to RxJs?